env.demo

```diff
@@ -69,3 +69,4 @@ DRF_THROTTLE_RATE_EVENTS_API='100000/day'
 DRF_THROTTLE_RATE_SEARCH_API='1000/day'
 #This allow access to the API with TokenAuthentication
 DRF_ALLOW_TOKENAUTHENTICATION=False
+ALLOW_REMOTE_USER_AUTH=True
```

Comment:
SET TO FALSE BY DEFAULT PLEASE

Reply:
C'est fait + Tests adaptés en conséquence